examples/proc_fieldSweep.py

```diff
@@ -1,142 +1,29 @@
-    (
-        "220217_5mM_TEMPOL_field_dep",
+with psd.figlist_var() as fl:
+    thisfile, exptype, nodename, post_proc, label_str = (
+        "240924_13p5mM_TEMPOL_field.h5",
         "ODNP_NMR_comp/field_dependent",
-        "Field_sweep",
-        "field_sweep_v1",  # in newer version files, we should be setting
-        #                  postproc_type as a top-level attribute of the
-        #                  nddata, and this should not be necessary
-        "TEMPOL field sweep",
-        (-250, 250),
-    ),
-]:
-    s = find_file(
+        "field_1",
+        "field_sweep_v4",
+        "240924 13.5 mM TEMPOL field sweep",
+    )
+    s = psd.find_file(
         thisfile,
-        exp_type=exp_type,
+        exp_type=exptype,
         expno=nodename,
-        postproc=postproc,
-        lookup=lookup_table,
-    )
-    # {{{Obtain ESR frequency and chunk/reorder dimensions
-    nu_B12_GHz = (s.get_prop("acq_params")["mw_freqs"][0]) / 1e9
-    # }}}
-    # {{{DC offset correction
-    s.ift("t2")
-    s.ift("ph1")
-    t2_max = s.getaxis("t2")[-1]
-    rx_offset_corr = s["t2" : (t2_max * 0.75, None)]
-    rx_offset_corr = rx_offset_corr.mean(["t2"])
-    s -= rx_offset_corr
-    s.ft(["ph1"])
-    s.ft("t2")
-    # }}}
-    # {{{ set up figure and plot raw data
-    fig, ax_list = subplots(1, 4, figsize=(10, 3))
-    fl.next("Field Sweep Processing", fig=fig)
-    fl.image(s, ax=ax_list[0])
-    ax_list[0].set_title("Raw data\nFrequency Domain")
-    # }}}
-    # {{{frequency filtering and phase correct
-    s = s["t2":(-1e3, 1e3)]
-    s.ift("t2")
-    best_shift = hermitian_function_test(select_pathway(s, signal_pathway))
-    s.setaxis("t2", lambda x: x - best_shift).register_axis({"t2": 0})
-    s = s["t2":(0, None)]
-    s["t2", 0] *= 0.5
-    s.ft("t2")
-    s = select_pathway(s, signal_pathway)
-    s /= zeroth_order_ph(s.C.mean("t2"))
-    nu_NMR = []
-    assert set(s.getaxis("indirect").dtype.names) == {
-        "Field",
-        "carrierFreq",
-    }, "'indirect' axis should be a structured array that stores the carrier frequency and the field"
-    # {{{ the following works fine for all the cases that we've tested, and so
-    # there's no need to change it.  But, after resolving everything, realized
-    # that the best way to deal with this would be to run a correlation on all
-    # the scans to determine the relative offsets, which would give both the
-    # relative offsets and the frequency shifts needed to align the signals, at
-    # the end.
-    all_offsets = zeros(len(s.getaxis("indirect")))
-    for z in range(len(s.getaxis("indirect"))):
-        fl.plot(s["indirect", z], ax=ax_list[1])
-        if z == 0:
-            ax_list[1].axvline(color="k", x=freq_slice[0])
-            ax_list[1].axvline(color="k", x=freq_slice[-1])
-        offset = s["indirect", z].C.mean("nScans").argmax("t2").item()
-        all_offsets[z] = offset
-        carrier_freq_MHz = s.getaxis("indirect")[z]["carrierFreq"]
```

Comment:
to use this, you would choose the option to use the carrier frequency rather than the field, and then, you could remove the final square bracket

In looking at this, I'm realizing that this loop can probably be vectorized, but we can deal with that after getting a loop that works.